tensorvis/tensorvisio/src/tensorvisiomodule.cpp

```diff
@@ -43,12 +43,15 @@
 #include <modules/tensorvisio/processors/vtkxmlrectilineargridreader.h>
 #include <modules/tensorvisio/processors/vtkstructuredpointsreader.h>
 #include <modules/tensorvisio/processors/vtkvolumereader.h>
+#include <modules/tensorvisio/util/vtkoutputlogger.h>
 
 namespace inviwo {
 
-TensorVisIOModule::TensorVisIOModule(InviwoApplication* app) : InviwoModule(app, "TensorVisIO") {   
-	registerProcessor<AmiraTensorReader>();
-	registerProcessor<NRRDReader>();
+TensorVisIOModule::TensorVisIOModule(InviwoApplication* app) : InviwoModule(app, "TensorVisIO") {
+    VtkOutputLogger::init();
```

Comment:
never deleted!

Reply:
Have you tried closing inviwo after implementing this? Might cause problems when the singleton is destroyed.

Reply:
good points

Reply:
Closing Inviwo is fine. However, unloading the module might cause some trouble.